src/components/card/index.tsx

```diff
@@ -1,48 +1,21 @@
 import React, { CSSProperties } from 'react';
-import { TextParentStyles } from 'src/index';
-
-interface ICardProps {
-  imgSrc?: string;
-  title: string;
-  text: string;
-  primaryBtnLabel?: string;
-  secondaryBtnLabel?: string;
-  disablePrimaryBtn?: boolean;
-  disableSecondaryBtn?: boolean;
-  onClickPrimaryBtn?: () => void;
-  onClickSecondaryBtn?: () => void;
-  imgStyle?: CSSProperties;
-  titleStyle?: CSSProperties;
-  cardStyle?: CSSProperties;
-  textStyle?: CSSProperties;
-  primaryBtnStyle?: CSSProperties;
-  primaryDisableBtnStyle?: CSSProperties;
-  secondaryBtnStyle?: CSSProperties;
-  secondaryDisableBtnStyle?: CSSProperties;
-  onClickView?: () => void;
-}
+import { EmbeddedManager, TextParentStyles } from 'src/index';
+import { EmbeddedMessageData } from '../types';
 
 /* Note: Add export to this const when support Embedded Message View Types in a later release. */
-const Card = (props: ICardProps) => {
+export const Card = (props: EmbeddedMessageData) => {
```

Comment:
`IterableEmbeddedMessage` instead of `EmbeddedMessagData`?

Reply:
We can't. `IterableEmbeddedMessage` is responsible for determining the message type received, while `EmbeddedMessageData` contains the properties needed for custom styling of Out-of-the-Box (OOTB) views.